build/.azure-pipelines.yml

```diff
@@ -11,13 +11,13 @@ pr:
 resources:
    containers:
      - container: windows
-       image: nventive/vs_build-tools:16.8.6
+       image: nventive/vs_build-tools:17.1.1
 
 variables:
 - name: NUGET_VERSION
-  value: 5.8.1
+  value: 6.1.0
 - name: VSTEST_PLATFORM_VERSION
-  value: 16.9.1
+  value: 17.1.1
```

Comment:
The version that you want to use it's [17.1.0](https://www.nuget.org/packages/Microsoft.TestPlatform/17.1.0)